.github/workflows/build.yml

```diff
@@ -9,6 +9,29 @@ on:
       - master
 jobs:
 
+  build-arm:
+    name: Build ESBMC with Boolector (ARM64)
+    runs-on: ['self-hosted', 'Linux', 'ARM64']
+    container: ubuntu:20.04
+    steps:
+     - name: cleanup old files
+       run: pwd && rm -rf ./*
```

Comment:
Each time you run, it's a new container, you shouldn't need to do this if you use `container:`.

Reply:
I am not sure, if the job finishes with an error I couldn't checkout the repo because of old files (from boolector). It looks like it only deletes the container if the job succedes. 

Maybe this is done so we can manually debug afterwards

Reply:
Ah, yes, that makes sense. :)